packages/flame/lib/src/components/mixins/notifiable.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flame/components.dart';
+import 'package:flame/game.dart';
+import 'package:meta/meta.dart';
+
+/// Makes a component capable of notify listeners of changes.
+///
+/// Notifieable components will automatically notify when
+/// new instances are added or removed to the game instance.
+///
+/// To notify internal changes of a component instance, the component
+/// should call [notifyListeners].
+mixin Notifiable on Component {
+  FlameGame get _gameRef {
```

Comment:
This could just be called `_game` I think, then it is ready for the "new standard".